src/main.rs

```diff
@@ -115,6 +115,18 @@ struct Args {
 	#[clap(long)]
 	expand_macros: bool,
 
+	/// Space or comma separated list of features to activate
```

Comment:
can features really be separated by comma?

Reply:
no, I had simple copy the message from cargo and forget about it

Reply:
well if cargo supports it than that's fine I guess

Reply:
Actually, if cargo says it's `Space or comma separated list of features to activate` then we should keep it that way. We don't parse the features anyways, just pass them through to cargo